docs/user/index.rst

```diff
@@ -123,11 +120,11 @@ Get a high-level overview of our platform:
 .. TODO: This next item needs its article to be finished in a separate PR
 .. https://github.com/readthedocs/readthedocs.org/pull/10071
 
-💡 :doc:`Continuous Documentation </integrations>`
-  Discover the advantages of having your documentation continuously deployed.
+💡 :doc:`/explanation/dedicated-platform`
+  An introduction to the reasons behind choosing a dedicated documentation platform such as Read the Docs.
 
-💡 :doc:`/choosing-a-site`
-  Learn about the differences between |org_brand| and |com_brand|.
+💡 :doc:`/explanation/continuous-documentation`
+  Discover the advantages of having your documentation continuously deployed.
 
 .. TODO: This next item needs its article to be finished in a separate PR
```

Comment:
I thought the goal of this PR was to finish the deep dive, and add it to the index?

Reply:
Thanks for fixing!